core/src/com/group66/game/BustaMove.java

```diff
@@ -4,42 +4,51 @@
 import com.badlogic.gdx.graphics.OrthographicCamera;
 import com.badlogic.gdx.graphics.g2d.SpriteBatch;
 import com.group66.game.screens.MainMenuScreen;
+import com.group66.game.settings.Config;
 
-public class BustaMove extends Game {
-	
-	//public static final int WIDTH = 400;
-	//public static final int HEIGHT = 600;
-	
-	// TODO make a bit nicer
-	public static final int WIDTH = 128 + 20;
-	public static final int HEIGHT = 220 + 20;
-	
-	public static final int BOUNCE_X_MIN = 10;
-	public static final int BOUNCE_X_MAX = WIDTH - 10;
-
-	public static final int BOUNCE_Y_MIN = 10;
-	public static final int BOUNCE_Y_MAX = HEIGHT - 10;
-
+/**
+ * The BustaMove main game class.
+ */
+public class BustaMove extends Game {	
 	
+	/** The batch. */
 	public SpriteBatch batch;
+	
+	/** The camera. */
 	public OrthographicCamera camera;
 	
+	/**
+	 * Gets the game height.
+	 *
+	 * @return the game height
+	 */
 	public int getGameHeight() {
-		return HEIGHT;
+		return Config.HEIGHT;
```

Comment:
This is done because our program now just runs in "windowed" mode. A future improvement can be to add view ports (I think) to keep the aspect ratio.
https://www.youtube.com/watch?v=Ml-XXQaN0hc
If you now resize the App, it all resizes (streches) but the "base" size is still the static Height and Width.

So basically this is the most extendable method for now to get everything working. Implementing the view port is a next step that can be taken to further improve the game (a next step)